### PR TITLE
Minor markups to align beta service with user research prototype

### PR DESCRIPTION
**Pull request by camillebaldock**
- Feedback link in BETA link was not marked as an external link
- Copy/design was not the same on title search and search results page as the prototype validated through user research, very minor changes

### CODE REVIEW

File: service/templates/search.html [hunk 1]
```diff
@@ -9,34 +9,20 @@
...
-
+                  <form action="/title-search/" method="POST" role="form" class="form">
+                      {{ form.hidden_tag() }}
+                      <h1 class="heading-xlarge form-title">Find a title</h1>
+                      <div class="form-group">
+                          <div class="text">
+                              <p class="lede"><label for="search_term">Search for the title of any property in England and Wales, even if you don’t own it.</label></p>
+                          </div>
+                          <p class="form-hint">For example a postcode, street name or title number</p>
+                          <input type="text" class="form-control" name="search_term" id="search_term">
```
Comment (lgonczar) [inline service/templates/search.html:20]: Are we deliberately losing the search term value?

Comment (camillebaldock) [inline service/templates/search.html:20]: We never access that search page with a prefilled value, so yes this is deliberate. I think the question you are raising is the one talked about earlier about validation and whether we should be getting back to the search page in some cases? I've tracked that as something to go over with Ian on Monday. lmk if I misunderstood your intent here :)

**Comment by aronjasper** [comment]
Looks good to me.

**Comment by camillebaldock** [comment]
Please merge at the same time as https://github.com/LandRegistry/digital-register-feeder/pull/93

File: service/templates/search.html (hunk 1 above)
Comment (lgonczar) [inline service/templates/search.html:20]: Oh, if we never return back to this page, then this is actually a fix :+1:

**Pull request merged**